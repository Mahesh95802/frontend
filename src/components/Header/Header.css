.main-panel-header{
    display: flex;
    justify-content: flex-start;
    align-items: center;
    max-height: 69px;
    align-items: center;
    font-weight: bolder;
    width: 100%;
    /* color: white; */
    font-size: 25px;
    padding: 20px;
}