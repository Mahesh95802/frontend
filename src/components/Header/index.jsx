import React from 'react';
import PropTypes from 'prop-types';

import './Header.css';

const Header = ({ title }) => {
    return (
        <div className="main-panel-header basic-padding">
            {title}
        </div>
    )
}

Header.propTypes = {
    title: PropTypes.string
}

export default Header;
