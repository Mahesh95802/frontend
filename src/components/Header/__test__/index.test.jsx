/* eslint-disable no-undef */
import { render } from '@testing-library/react';
import React from 'react';
import Header from '..';

describe('Header', () => {
    it('should render correctly', () => {
        const { asFragment } = render(<Header title={'Hello World!!'}/>);
        expect(asFragment()).toMatchSnapshot();
    });
});