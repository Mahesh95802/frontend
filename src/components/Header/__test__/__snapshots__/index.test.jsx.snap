// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="main-panel-header basic-padding"
  >
    Hello World!!
  </div>
</DocumentFragment>
`;
